apps/server/src/routes/files/index.ts

```
@@ -1,208 +1,349 @@
+} from "@nimbus/cache/rate-limiters";
+import type { ApiResponse, UploadedFile } from "@/routes/types";
+import type { File } from "@/providers/interface/types";
 import { TagService } from "@/routes/tags/tag-service";
-import type { File } from "@/providers/google/types";
-import { getAccount } from "@/lib/utils/accounts";
-import type { ApiResponse } from "@/routes/types";
+import { getDriveManagerForUser } from "@/providers";
+import { securityMiddleware } from "@/middleware";
+import { type Session } from "@nimbus/auth/auth";
 import type { Context } from "hono";
+import { Readable } from "stream";
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)